Installer/source/main.cpp

```diff
@@ -1086,194 +1209,238 @@ s8 WritePriiloader( bool priiloader_found )
 	printf("Writing Priiloader app...");
 	gprintf("Writing Priiloader");
 
-	char temp_dest[ISFS_MAXPATH];
-	memset(temp_dest,0,ISFS_MAXPATH);
-	char *ptemp = strstr(original_app,"/");
-	while(ptemp != NULL && strstr(ptemp+1,"/") != NULL)
-	{
-		ptemp = strstr(ptemp+1,"/");
-	}
-	if(ptemp == NULL)
+	if (is_vwii)
 	{
-		gprintf("error WritePriiloader: failed to compile tmp filepath",fd);
-		abort("\nFailed to compile tmp filepath");
-	}
-	if(ptemp[0] == '/')
-	{
-		ptemp = ptemp+1;
-	}
-	sprintf(temp_dest,"/tmp/%s",ptemp);
-	ISFS_Delete(temp_dest);
-	ret = ISFS_CreateFile(temp_dest,SysPerm.attributes,SysPerm.ownerperm,SysPerm.groupperm,SysPerm.otherperm);
-    if (ret < 0)
-    {
-        gprintf("error %d", fd);
-        abort("\nFailed to create file for Priiloader");
-    }
+		// Write priiloader to an additional content file for vWii (2XXXXXXX.app)
+		ret = nand_copy(additional_app, (u8 *)priiloader_app,priiloader_app_size,SysPerm);
+		if (ret < 0)
+		{
+			ISFS_Delete(additional_app);
+			abort("\nUnable to replace the System Menu. error %d",ret);
+		}
+		else
+		{
+			gprintf("System Menu replaced.");
+			printf("Done!\r\n");
+		}	
 
-	fd = ISFS_Open(temp_dest,ISFS_OPEN_RW);
-	if (fd < 0)
-	{
-		gprintf("error %d",fd);
-		abort("\nFailed to open file for Priiloader writing");
+		// On vWii we replace the SM boot content with BC-NAND
+		// TODO don't hardcode the .app file, read boot index from BC-NAND tmd?
+		printf("Replacing System Menu app...");
+		ret = nand_copy("/title/00000001/00000200/content/00000003.app",original_app,SysPerm);

```

Comment:
> 1[id] = SM (so priiloader & tools need no changes)

Unfortunately we cannot do that since BC-NAND will boot from content 1.
That's why the already existing content 1 had to be moved to the end (now content 9).